library/src/main/java/com/fraggjkee/smsconfirmationview/SmsConfirmationView.kt

```diff
@@ -162,7 +223,16 @@ class SmsConfirmationView @JvmOverloads constructor(
         }
     }
 
+    // TODO (BA, 5/18/23): `ACTION_MULTIPLE`/`event.characters` deprecated
+    //  in API 29 noting "No longer used by the input system", though still working on API 33
+    //  and without alternatives from Google.
+    @Suppress("DEPRECATION")
     private fun handleKeyEvent(keyCode: Int, event: KeyEvent): Boolean = when {
+        event.action == KeyEvent.ACTION_MULTIPLE && event.keyCode == KeyEvent.KEYCODE_UNKNOWN && allowCodePaste -> {
```

Comment:
haven't checked it by myself yet, but wondering if we can somehow use any of those functions instead:
- [View::onReceiveContent](https://developer.android.com/reference/android/view/View#onReceiveContent(android.view.ContentInfo))
- [View::setOnReceiveContentListener](https://developer.android.com/reference/android/view/View#setOnReceiveContentListener(java.lang.String[],%20android.view.OnReceiveContentListener))
- [ViewCompat::setOnReceiveContentLisnter](https://developer.android.com/reference/androidx/core/view/ViewCompat#setOnReceiveContentListener(android.view.View,java.lang.String[],androidx.core.view.OnReceiveContentListener))